configs/cms-xcache/overrides/systemd/10-cms-refresh-proxy-overrides.conf

```
@@ -0,0 +1,2 @@
+[Service]
+ExecStart = /usr/libexec/xcache/renew-proxy cms
```

I'm pretty sure you need to clear the `ExecStart` attr if you're overriding it

```suggestion
ExecStart =
ExecStart = /usr/libexec/xcache/renew-proxy cms
```